server/main.lua

```diff
@@ -148,7 +151,7 @@ end)
 
 
 ---@param employeeCitizenId string
----@param boss Player
+---@param boss table The player object
```

Comment:
```suggestion
---@param boss Player
```
Should keep this as is and can optionally add a disable annotation